src/main/java/no/nav/fo/veilarbregistrering/db/arbeidssoker/ArbeidssokerRepositoryImpl.java

```diff
@@ -52,30 +59,47 @@ private long nesteFraSekvens(String sekvensNavn) {
         return jdbcTemplate.queryForObject("select " + sekvensNavn + ".nextval from dual", Long.class);
     }
 
+    @Override
+    public Arbeidssokerperioder finnFormidlingsgrupper(Bruker bruker) {
+        Arbeidssokerperioder arbeidssokerperioder = finnFormidlingsgrupper(bruker.getGjeldendeFoedselsnummer());
+
+        List<Arbeidssokerperioder> historiskeArbeidssokerperioder = bruker.getHistoriskeFoedselsnummer().stream()
+                .map(this::finnFormidlingsgrupper)
+                .collect(toList());
+
+        return arbeidssokerperioder.slaaSammenMed(historiskeArbeidssokerperioder);
+    }
+
     @Override
     public Arbeidssokerperioder finnFormidlingsgrupper(Foedselsnummer foedselsnummer) {
         String sql = "SELECT * FROM FORMIDLINGSGRUPPE WHERE FOEDSELSNUMMER = ?";
 
         List<ArbeidssokerperiodeRaaData> arbeidssokerperioder = jdbcTemplate.query(
                 sql,
-                new Object[]{foedselsnummer.stringValue()},
-                (rs, row) -> new ArbeidssokerperiodeRaaData(
-                        rs.getString("FORMIDLINGSGRUPPE"),
-                        rs.getTimestamp("FORMIDLINGSGRUPPE_ENDRET")
-                )
+                new Object[]{foedselsnummer.stringValue()}, new ArbeidssokerperiodeRaaDataRowMapper()
         );
 
         return map(arbeidssokerperioder);
     }
 
     @Override
-    public Arbeidssokerperioder finnFormidlingsgrupper(Bruker bruker) {
-        Arbeidssokerperioder arbeidssokerperioder = finnFormidlingsgrupper(bruker.getGjeldendeFoedselsnummer());
+    public Arbeidssokerperioder finnFormidlingsgrupper(List<Foedselsnummer> foedselsnummer) {
```

Comment:
Skal vi ta denne i bruk i den nye `hentArbeidssokerperioder()` i servicen med en gang? Og samtidig fjerne `finnFormidlingsgrupper(Bruker bruker)` og `slaaSammenMed()`.

Reply:
Ja - jeg er i gang. Men tenkte jeg skulle merge denne først, for å ikke gjøre for mange ting samtidig.